WikiWatershed/model-my-watershed
# Remove back buttons

## Overview

The various back buttons in the app have been superseded by the new
breadcrumbs feature, and are no longer needed.

The back button DOM elements, and related style rules and code, are
removed here.

Connects #1726

### Demo

![image](https://cloud.githubusercontent.com/assets/1042475/23713341/9dd0ed46-03f3-11e7-981f-b335ba8fbf41.png)
*Returning to Draw from Analyze*

![image](https://cloud.githubusercontent.com/assets/1042475/23713358/aed5721a-03f3-11e7-9edf-ee879c84f3eb.png)
*Modeling page*

![image](https://cloud.githubusercontent.com/assets/1042475/23713386/be084c62-03f3-11e7-9e4e-0bbc18c5ebd3.png)
*Compare page*

## Testing Instructions

* Start the app server and rebundle all of the JS bundles.
* Navigate back and forth through the app, and verify that all of the back buttons have been removed, and that the main area of the app is always sized correctly (i.e. there isn't a blank space where the analyze -> draw footer used to be)
